study-builder/studies/rarex/quality-of-life.conf

```diff
@@ -0,0 +1,1650 @@
+{
+  include required("../snippets/activity-general-form.conf"),
+  "studyGuid": "RAREX",
+  "activityCode": "QUALITY_OF_LIFE",
+  "versionTag": "v1",
+  "displayOrder": 6,
+  "writeOnce": false,
+  "maxInstancesPerUser": 1,
+  "translatedNames": [
+    {
+      "language": "en",
+      "text": ${i18n.en.qol_name}
+    }
+  ],
+  "translatedTitles": [
+    {
+      "language": "en",
+      "text": ${i18n.en.qol_title}
+    }
+  ],
+  "translatedSummaries": [
+    {
+      "statusCode": "CREATED",
+      "language": "en",
+      "text": ${i18n.en.qol_summary_created}
+    },
+    {
+      "statusCode": "IN_PROGRESS",
+      "language": "en",
+      "text": ${i18n.en.qol_summary_in_progress}
+    },
+    {
+      "statusCode": "COMPLETE",
+      "language": "en",
+      "text": ${i18n.en.qol_summary_complete}
+    }
+  ],
+  "sections": [
+    {
+      "nameTemplate": null,
+      "icons": [],
+      "blocks": [
+        {
+          "titleTemplate": null,
+          "bodyTemplate": {
+            "templateType": "HTML",
+            "templateText": """<h2 class="subtitle subtitle--light">$qol_intro</h2>""",
+            "variables": [
+              {
+                "name": "qol_intro",
+                "translations": [
+                  {
+                    "language": "en",
+                    "text": ${i18n.en.qol_intro}
+                  }
+                ]
+              }
+            ]
+          },
+          "blockType": "CONTENT",
+          "shownExpr": null
+        },
+        {
+          "question": {
+            include required("../snippets/picklist-question-single-list.conf"),
+            "stableId": "QOL_OVERALL_HEALTH"
+            "hideNumber": true,
+            "promptTemplate": {
+              "templateType": "HTML",
+              "templateText": """
+                      <p>$qol_overall_health</p>
+                  """,
+              "variables": [
+                {
+                  "name": "qol_overall_health",
+                  "translations": [
+                    {
+                      "language": "en",
+                      "text": ${i18n.en.qol_overall_health}
+                    }
+                  ]
+                }
+              ]
+            },
+            "picklistOptions": [
+              {
+                "stableId": "QOL_OVERALL_HEALTH_VERY_POOR",
```

Comment:
Picklist option stable ids are scoped to the question, so having the `QOL_OVERALL_HEALTH` prefix is a bit redundant. If we remove it, answers in data export will also show up a little nicer.

Reply:
Fixed where applicable